README.md

```diff
@@ -107,13 +107,36 @@ Without a from name, your sending email will be the name of the email. Use the o
 
 ```lisp
 (sendgrid:send-email :to "recipient@example.com"
-                     :from "noreply@example.com"
-					 :from-name "Jane Doe"
-                     :subject "Sending emails from SendGrid is fun!"
-                     :content-type "text/html"
-                     :content "<h1>A title</h1><br/><strong>Sending emails from SendGrid is fun!</strong>")
+	              :from "noreply@example.com"
+		      :from-name "Jane Doe"
+		      :subject "Sending emails from SendGrid is fun!"
+		      :content-type "text/html"
+		      :content "<h1>A title</h1><br/><strong>Sending emails from SendGrid is fun!</strong>")
 ```
+## Email scheduling - :send-at key
+
+The :send-at key takes a unix epoch time and sends your email at that specific time.
+ 
+Free sendgrid accounts (at this time - 4th Oct 23) are limited to sending up to 4 days in the future. Therefore the `:send-at` key is limited to sending emails between 1-4 days in the future.
```

Comment:
now that the api allows 3 days and not 4, the README should be changed (here and below).

Reply:
Good spot.